SU-HSDO/suhumsci
# HSD8-744 fix menu keyboard

# READY FOR REVIEW

# Summary
- Fix keyboard navigation on main menu

# Need Review By (Date)
- asap

# Urgency
- high

# Steps to Test
1. checkout this branch
1. clear caches
1. navigate the main menu and ensure you can see where you are and you can click (enter) on items.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
